pkg/build/builder/util.go

```diff
@@ -1,7 +1,11 @@
 package builder
 
 import (
-	"github.com/openshift/origin/pkg/build/api"
+	"github.com/golang/glog"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	buildutil "github.com/openshift/origin/pkg/build/util"
+	osclient "github.com/openshift/origin/pkg/client"
```

Comment:
that's a ton of places, i think i'd rather do that in a separate PR given the way this one is going.